### PR TITLE
This is just a suggestion to migrate to standalone components etc

### PR DESCRIPTION
**Pull request by Julian-B90**
- Migrate Lib to Standalone API
- Update Cypress 13.13.0

### CODE REVIEW

**Comment by sibiraj-s** [comment]
Thanks, Will look into it. I would have preferred a discussion prior to making breaking changes.

**Comment by Julian-B90** [comment]
Maybe with that https://github.com/sibiraj-s/ngx-tiptap/pull/70/commits/2eb86c0a303c2fd42630eaff466fbe37bba937e3 we can offer both?

**Comment by sibiraj-s** [comment]
> Maybe with that [2eb86c0](https://github.com/sibiraj-s/ngx-tiptap/commit/2eb86c0a303c2fd42630eaff466fbe37bba937e3) we can offer both?

I am not sure at this point if there would be any size increase if both are provided, since I never tried it.

Also, its fine to make breaking changes for a better API or make it more easy to use. I just said I wish there could have been a discussion nothing else.

File: projects/ngx-tiptap/src/lib/ngx-tiptap.module.ts
Comment (sibiraj-s) [inline projects/ngx-tiptap/src/lib/ngx-tiptap.module.ts:1]: Can you delete this file, we can proceed with just standalone components, I can make a semver major release.

File: projects/ngx-tiptap/src/public-api.ts [hunk 1]
```diff
@@ -8,7 +8,7 @@ export * from './lib/bubble-menu.directive';
 export * from './lib/draggable.directive';
 export * from './lib/node-view-content.directive';
 export * from './lib/node-view.component';
-export * from './lib/ngx-tiptap.module';
+// export * from './lib/ngx-tiptap.module';
```
Comment (sibiraj-s) [inline projects/ngx-tiptap/src/public-api.ts:11]: This line can be removed.

**Review by sibiraj-s — CHANGES_REQUESTED** [review]
Thanks for the PR ❤️ 

Changes looks good to me, can you delete the module file? I will test these changes a bit and release them.

**Comment by sibiraj-s** [comment]
Sorry for the delay, If you could resolve the conflicts, I can get it merged and released as next major.

**Comment by Julian-B90** [comment]
@sibiraj-s i have resolve the merge conflicts

**Comment by Julian-B90** [comment]
@sibiraj-s ?

**Review by sibiraj-s — APPROVED** [review]

**Pull request merged**